src/modules/visits/controllers/visits.controller.ts

```diff
@@ -226,7 +226,10 @@ export class VisitsController {
 		@SessionUser() user: SessionUserEntity
 	): Promise<Visit | null> {
 		// Check if the user is a CP admin or a Kiosk user for the requested space
-		if (visitorSpaceSlug === user.getVisitorSpaceSlug()) {
+		if (
+			visitorSpaceSlug === user.getVisitorSpaceSlug() ||
+			user.getGroupId() === GroupId.MEEMOO_ADMIN
```

Comment:
At some point we would like to get rid of the GroupId enum since it contains uuids and we don't like hardcoding uuids in the code. So prefer GroupName where possible.
```suggestion
			user.getGroupName() === GroupName.MEEMOO_ADMIN
```

Maybe you can also add a depracted notice to the GroupId enum, so it warns other devs to not use it anymore:
```
/**
 * @deprecated At some point we would like to get rid of the GroupId enum since it contains uuids and we don't like hardcoding uuids in the code. So prefer GroupName where possible.
 */
export enum GroupId...
```